src/styled/nav-button.ts

```diff
@@ -0,0 +1,10 @@
+import styled from '@emotion/styled';
+import { size } from './theme';
+
+const StyledNavList = styled('ul')`
```

Comment:
Should this be a `ul` tag? It looks like it just wraps the individual nav buttons, and it is causing lighthouse to fail because it expects only a `li` tag under `ul`.